pkg/provider/azure_controller_common.go

```diff
@@ -337,9 +350,38 @@ func (c *controllerCommon) attachDiskBatchToNode(ctx context.Context, subscripti
 			}
 		}
 
-		err = vmset.WaitForUpdateResult(resultCtx, future, resourceGroup, "attach_disk")
+		backoffManager := wait.Backoff{
+			Duration: updateVMUpdateRetryDuration,
+			Factor:   updateVMUpdateRetryFactor,
+			Steps:    updateVMUpdateRetrySteps,
+		}
```

Comment:
Consider making `backoffManager` a global variable instead of having separate constants for the parameters. For example, see [`defaultBackoff`](https://github.com/kubernetes-sigs/cloud-provider-azure/blob/d1090eb6ae7493cb0e2492242b49e895b026c9b7/pkg/provider/azure_controller_common.go#L68).

Reply:
Nevermind, I see the `Step` function mutates the struct, so using a global variable would be innapropriate.